.github/actions/run-python-test-set/action.yml

```diff
@@ -179,7 +179,15 @@ runs:
 
         # Wake up the cluster if we use remote neon instance
         if [ "${{ inputs.build_type }}" = "remote" ] && [ -n "${BENCHMARK_CONNSTR}" ]; then
-          ${POSTGRES_DISTRIB_DIR}/v${DEFAULT_PG_VERSION}/bin/psql ${BENCHMARK_CONNSTR} -c "SELECT version();"
+          QUERIES=("SELECT version()")
+          if [[ "${PLATFORM}" = "neon"* ]]; then
+            QUERIES+=("SHOW neon.tenant_id")
+            QUERIES+=("SHOW neon.timeline_id")
+          fi
+
+          for q in "${QUERIES[@]}"; do
+            ${POSTGRES_DISTRIB_DIR}/v${DEFAULT_PG_VERSION}/bin/psql ${BENCHMARK_CONNSTR} -c "${q}"
+          done
```

Comment:
I think we want to avoid running three psql subprocesses, which establish 3 connections (3 times handshake, going thru proxy etc.) if we can run the same commands in one connection

Reply:
Unfortunately, it prints only the last statement results:
https://github.com/neondatabase/neon/actions/runs/9762728193/job/26946868145#step:6:249

```
+ /tmp/neon/pg_install/v14/bin/psql '***' -c 'SELECT version();SHOW neon.tenant_id;SHOW neon.timeline_id;'
         neon.timeline_id         
----------------------------------
 a7122099cdf881728c8bf02cefef0e0a
(1 row)
```

I'd say we're ok to run 3 queries instead of one, given we do it once before the job